### PR TITLE
fix https://github.com/webdriverio/selenium-standalone/issues/653

### PR DESCRIPTION
**Pull request by wyhmichael**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "selenium-standalone",
-  "version": "8.0.10",
+  "version": "8.0.11",
```
Comment (christian-bromann) [inline package.json:3]: We have an automated way to update the version
```suggestion
  "version": "8.0.10",
```

**Review by christian-bromann — CHANGES_REQUESTED** [review]
Can we add a test for Selenium 4?

**Comment by wyhmichael** [comment]
> Can we add a test for Selenium 4?

Thanks @christian-bromann  I reverted the version update

And there's tests for Selenium 4, and looks good already: https://github.com/webdriverio/selenium-standalone/pull/654/files#diff-96299f8ea73dbd04f0260ab3ae2db9a749fbd43498bd9733807d2c40405d9bf6L65-R108, do you want more tests around it?

**Review by christian-bromann — APPROVED** [review]
Ah ok, in this case this looks good to me 👍 

Thanks for the contribution!

**Comment by wyhmichael** [comment]
> Ah ok, in this case this looks good to me 👍
> 
> Thanks for the contribution!

Thanks @christian-bromann for confirmation!

Sorry that I had a linting error, would you pls take another look?

**Pull request merged**